src/components/Agenda/index.tsx

```diff
@@ -26,20 +33,66 @@ const compareByDateTime = (a: AgendaItem, b: AgendaItem) =>
  * and list of calendar events
  */
 
-const Agenda = (): ReactElement => {
-  const account = useContext(AccountContext)
+const REFRESH_INTERVAL = 10000
 
+const Agenda = (): ReactElement => {
+  const accountContext = useContext(AccountContext)
   const events: AgendaItem[] = useMemo(
     () =>
-      account.calendars
-        .flatMap((calendar) =>
-          calendar.events.map((event) => ({ calendar, event })),
-        )
+      accountContext.account.calendars
+        .flatMap((calendar) => {
+          return calendar.events.map((event) => ({ calendar, event }))
+        })
         .sort(compareByDateTime),
-    [account],
+    [accountContext.account],
+  )
+
+  const [currentHour, setCurrentHour] = useState(DateTime.local().hour)
+  useEffect(() =>
+    runEvery(REFRESH_INTERVAL, () => {
+      setCurrentHour(DateTime.local().hour), []
+    }),
```

Comment:
Move dependency array so that it's the second argument to useEffect. 

This code will just run once.